assets/blueprints/blueprint.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"landingPage": "/wp-admin/profile.php",
+	"preferredVersions": {
+		"php": "7.4",
+		"wp": "latest"
+	},
+	"steps": [
+		{
+			"step": "login"
+		},
+		{
+			"step": "installPlugin",
```

Comment:
WP.org might be doing this out of the box but we keep it here to ensure we can test it without deploying changes to WP.org.